[17] Socialie/Socialie/Views/Home View/HomeView.swift

```diff
@@ -9,46 +9,52 @@ import SwiftUI
 
 struct HomeView: View {
 	
+	/// Helps selecting the layout of `HomeView`
+	private enum Layout {
+		case asGrid
+		case asList
+	}
+	
 	@StateObject
 	private var vm: Self.ViewModel
 	
+	@State
+	private var selectedLayout: Layout
+	
 	init(vm: Self.ViewModel = .init()) {
 		_vm = StateObject(wrappedValue: vm)
+		selectedLayout = .asGrid
 	}
 	
-	@State
-	private var isGridLayout: Bool = true
-	
     var body: some View {
 		NavigationView {
 			Group {
-				if isGridLayout {
-					HomeGridView(viewModel: vm)
-				} else {
-					HomeListView(viewModel: vm)
+				switch selectedLayout {
+					case .asGrid: HomeGridView(viewModel: vm)
+					case .asList: HomeListView(viewModel: vm)
```

Comment:
I like this change. It makes the layout switching more user-friendly and avoid confusion